package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  boost,
+  cmake,
+  cxxopts,
+  eigen,
+  #fetchFromGitHub,
+  glew,
+  gtest,
+  lib,
```

Comment:
What's this?

Reply:
This is "Nixpkgs lib", ref. https://github.com/NixOS/nixpkgs/blob/master/lib/README.md